src/components/SalesRepTable/SalesRepTableRow.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledTableRow = styled.tr`
+  background-color: ${(props) =>
+    props.index % 2 === 0 ? '#eaeaea6e' : 'white'};
+  transition: all 0.05s ease-in-out;
+  :hover {
+    background-color: #c9d5e7;
+  }
+`;
+
+const StyledTableCell = styled.td`
+  padding: 0.5rem 1rem 0.5rem 1rem;
+  text-align: right;
+  color: #3d3d4a;
+  font-size: 0.8rem;
+`;
+
+const SalesRepTableRow = ({ item, index }) => (
+  <StyledTableRow index={index}>
+    <StyledTableCell
+      style={{
+        textAlign: 'left',
+        backgroundColor: '#eaeaea',
+        color: 'black',
+        fontWeight: 600,
+      }}
```

Comment:
Personally, I would extend from `StyledTableCell`. for example `const TomatoButton = styled(Button)`, alternative you can use `:first` `:nth-child` on the `StyledTableRow` to style the `td` element.